### PR TITLE
Update production capacity RO.yaml

### PR DESCRIPTION
**Pull request by RobertH-prog**
Updated production capacity for 2024 from Transelectrica ( national grid operator) source .

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [ ] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

**Review by VIKTORVAV99 — COMMENTED** [review]
Hi and thanks for the PR!

Is this using the same source as listed in `DATA_SORUCES.md`?

If it is we can merge this right away, otherwise this file also needs to be updated. 🙂

**Comment by RobertH-prog** [comment]
Yes, I added the new source and the link for it in the file. The data comes from national grid company (Transelectrica), they release regular updates on the capacity installed and you can find it in the link at section 20.

**Comment by RobertH-prog** [comment]
Deleted expired source and added the latest one in `DATA_SORUCES.md` .

File: config/zones/RO.yaml [hunk 1]
```diff
@@ -309,4 +309,6 @@ sources:
   : link:
       https://unece.org/sites/default/files/2022-04/LCA_3_FINAL%20March%202022.pdf#page=37,
       https://proceedings.windeurope.org/biplatform/rails/active_storage/disk/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaDdDRG9JYTJWNVNTSWhORFJ0ZDJJMWVUbG9OMll6TVRaaGEza3lkamgxZG1aM056WnZZZ1k2QmtWVU9oQmthWE53YjNOcGRHbHZia2tpQVk1cGJteHBibVU3SUdacGJHVnVZVzFsUFNKWGFXNWtaWFZ5YjNCbExWZHBibVF0Wlc1bGNtZDVMV2x1TFVWMWNtOXdaUzB5TURJeExYTjBZWFJwYzNScFkzTXVjR1JtSWpzZ1ptbHNaVzVoYldVcVBWVlVSaTA0SnlkWGFXNWtaWFZ5YjNCbExWZHBibVF0Wlc1bGNtZDVMV2x1TFVWMWNtOXdaUzB5TURJeExYTjBZWFJwYzNScFkzTXVjR1JtQmpzR1ZEb1JZMjl1ZEdWdWRGOTBlWEJsU1NJVVlYQndiR2xqWVhScGIyNHZjR1JtQmpzR1ZBPT0iLCJleHAiOiIyMDIyLTExLTAyVDE1OjU0OjAzLjEyNFoiLCJwdXIiOiJibG9iX2tleSJ9fQ==--c25280a7345257bd91bfaf6d64ddb75c55eef799/Windeurope-Wind-energy-in-Europe-2021-statistics.pdf?content_type=application%2Fpdf&disposition=inline%3B+filename%3D%22Windeurope-Wind-energy-in-Europe-2021-statistics.pdf%22%3B+filename%2A%3DUTF-8%27%27Windeurope-Wind-energy-in-Europe-2021-statistics.pdf
+  Transelectrica:
+    link: https://www.transelectrica.ro/en/web/tel/productie
```
Comment (VIKTORVAV99) [inline config/zones/RO.yaml:313]: This can be removed, it was only really intended for emission sources.
```suggestion
```

**Review by VIKTORVAV99 — COMMENTED** [review]
One minor thing then we can merge this!

Thanks for doing this!

**Comment by RobertH-prog** [comment]
Ok, deleted the source. I didn't knew it was only for emission sources. Thanks for letting me know.

**Pull request merged**